source/recurring_payments/index.html.md.erb

```diff
@@ -209,6 +215,15 @@ If a recurring payment fails when we try to take it, it’s handled like any oth
 
 Read more about creating payments in [our guidance for taking payments](/making_payments) or our [API reference page for creating payments](/api_reference/create_a_payment_reference).
 
+### Use GOV.UK Pay's optional features with recurring payments
```

Comment:
As I'm not looking at the whole doc I wanted to check the heading levels were as you wanted.